From ac184811313344d6ad0276ba5aaf18c286fe0586 Mon Sep 17 00:00:00 2001
From: Dinh Nguyen <dinh.nguyen@intel.com>
Date: Fri, 20 Oct 2017 18:36:36 -0500
Subject: [PATCH 02/25] arm64: dts: stratix10: Use PSCI node instead of
 spin-table for SMP

commit  61bf26dad85b85cc44e2dfef1d26897e167aeaae from
https://github.com/altera-opensource/linux-socfpga.git

This reverts commit e6c041984533 ("FogBugz #491251: temporary remove
 PSCI node")
U-Boot has implemented PSCI which is preferable to the spin-table
so go back to using the PSCI node.

Signed-off-by: Dinh Nguyen <dinguyen@kernel.org>
Signed-off-by: Thor Thayer <thor.thayer@linux.intel.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm64/boot/dts/altera/socfpga_stratix10.dtsi | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/arch/arm64/boot/dts/altera/socfpga_stratix10.dtsi b/arch/arm64/boot/dts/altera/socfpga_stratix10.dtsi
index 8ea5dae..5dd4bcd 100644
--- a/arch/arm64/boot/dts/altera/socfpga_stratix10.dtsi
+++ b/arch/arm64/boot/dts/altera/socfpga_stratix10.dtsi
@@ -69,6 +69,11 @@
 				     <&cpu3>;
 	};
 
+	psci {
+		compatible = "arm,psci-0.2";
+		method = "smc";
+	};
+
 	intc: intc@fffc1000 {
 		compatible = "arm,gic-400", "arm,cortex-a15-gic";
 		#interrupt-cells = <3>;
-- 
2.8.1

